integration-libs/cdc/manage-users/manage-users.module.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import {
+  CdcB2BUserService,
+  CdcUserListService,
+} from '@spartacus/cdc/manage-users';
+import { UserListService } from '@spartacus/organization/administration/components';
+import { B2BUserService } from '@spartacus/organization/administration/core';
+
+@NgModule({
+  providers: [
+    { provide: B2BUserService, useClass: CdcB2BUserService },
+    { provide: UserListService, useClass: CdcUserListService },
+  ],
+})
+export class ManageUsersModule {}
```

Comment:
Let's prefix this module with `Cdc`, as it's an "entry-point" module imported in the customer's app. So let's make it verbose it's a `Cdc`-specific module.

Moreover please, add a prefix similarly:
- `cdc-` to filename
- `CDC_` to constants (e.g. in schematics) - `MANAGE_USERS_MODULE` -> `CDC_MANAGE_USERS_MODULE`

What do you think about renaming the phrase "ManageUsers" to "CdcOrganizationAdministration"? (to follow similar naming convetions as we took when extending sub-libraries UserProfile and UserAccount.

Since we're extending the original entry point `@spartacus/organization/administration`, I suggest rename this module rather to `CdcOrganizationAdministration` (and accordingly: filename `cdc-organization-administration.module` and constants in schematics `CDC_ORGANIZATION_ADMINISTRATION`). Also let's rename the CDC library's sub-entry-point from `@spartacus/cdc/manage-users` to `@spartacus/cdc/organization-administration`.

PS. Another benefit of this renaming is that we can put here possibly more extensions of the original OrganizationAdministration module - e.g. disabling the default UI parts in routes of UserGroups and Units - not to allow for modifying an user from those views.

Reply:
done